README.md

```diff
@@ -109,6 +109,22 @@ export RCL_LOGGING_IMPLEMENTATION=rcl_logging_syslog
 colcon build --symlink-install --cmake-clean-cache --packages-select rcl_logging_syslog rcl
 ```
 
+### Test
+
+Following to the previous build procedure, we need to copy [ros2-test.conf](./config/ros2-test.conf) to `/etc/rsyslog.d/`, and restart the `rsyslog`.
+
+```bash
+sudo cp ./config/ros2-test.conf /etc/rsyslog.d/
+```
+
+Then we can start the `colcon test`:
+
+This test creates and uses `/var/log/rcl_logging_syslog` directory in temporary for the test, and it checks the file existence and contents using `rcl_logging_syslog`.
+
+```bash
+colcon test --event-handlers console_direct+ --packages-select rcl_logging_syslog
```

Comment:
this actually does not require to build rcl, because test uses rcl_logging_syslog interface directly. but i guess that is something nobody cares at this procedure.